### PR TITLE
Seeding Data

### PR DESCRIPTION
**Pull request by sienna-kopf**
# SOM-Timer BE Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- Figuring out how to seed real data to the DB for timers and exercises. Data for exercises pulled from [this form](https://docs.google.com/spreadsheets/d/18u89AJRy0iTrD8Z4lMkhYACbhokARcupufcJbg8YQrE/edit#gid=0)
- Run seeds after seting up using `flask db_seed` 
- You should see a confirmation message "Database Seeded!" if seeding is successful. 
- Tested in postman to ensure that DB is being populated. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [ ] Yes
- [x] In postman

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs/pry sessions
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

# Additionally, I tried to move the functionality into an existing file (manage.py) without luck.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from application import create_app, db
+from application.models.exercise import Exercise
+from application.models.timer import Timer
+from flask_sqlalchemy import SQLAlchemy
+import os
+
+app = create_app("development")
+db = SQLAlchemy(app)
+
+@app.cli.command('db_seed')
```
Comment (sciencefixion) [inline app.py:11]: This is some really friendly syntax! I'm excited to try this out!

**Review by sciencefixion — APPROVED** [review]

**Review by rwilliams659 — APPROVED** [review]
Great job! Left some thoughts on categorizing videos by duration but definitely open to other ideas - there's definitely no clear answer on the best way to do it!

File: requirements.txt [hunk 2]
```diff
@@ -8,3 +8,4 @@ nose-cov
 psycopg2
 six
 flask-testing
+coverage
```
Comment (Chulstro) [inline requirements.txt:11]: Oh interesting, coverage is done through the requirements?

**Review by Chulstro — APPROVED** [review]

**Pull request merged**